### PR TITLE
feat: remove redundant organisation code and search index

### PR DESCRIPTION
**Pull request by jrabbott**
### Context
[AB#206189](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/206189) ([AB#206781](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/206781))

### Change proposed in this pull request
Removes redundant organisation code and search index

### Guidance to review 
None

### Checklist
- [x] Work items have been linked [(use AB#)](https://learn.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-to-link-from-github-to-azure-boards-work-items)
- [x] Your code builds clean without any errors or warnings
- [x] You have run all unit/integration tests and they pass
- [x] Your branch has been rebased onto main
- [x] You have tested by running locally

### CODE REVIEW

File: documentation/developers/features/find-organisation.md [hunk 1]
```diff
@@ -63,19 +62,17 @@ A([✏️ Enter search criteria]) --> B[GET\n/api/establishments/suggest?type=sc
 
 On the Web side, `GET /api/establishments/suggest` proxies to the Establishment API based on the `type` in the query string:
```
Comment (WolfyUK) [inline documentation/developers/features/find-organisation.md:63]: It looks like this URL has changed

Comment (jrabbott) [reply documentation/developers/features/find-organisation.md:63]: Good spot

**Comment by WolfyUK** [comment]
This all looks fine to me, but I can't spin up locally to validate right now due to other refactoring in progress. If this is urgent then I can approve right now, otherwise it may be a little bit before I can click the button.

**Comment by jrabbott** [comment]
@WolfyUK I'll let you know when it becomes a blocker.

**Review by WolfyUK — DISMISSED** [review]
Validated locally 👍🏼

File: documentation/developers/features/find-organisation.md (hunk 1 above)
Comment (jrabbott) [reply documentation/developers/features/find-organisation.md]: Updated and pushed

**Review by WolfyUK — APPROVED** [review]

**Pull request merged**